Add LPWAN definition (#13)
DataHighway-DHX/whitepaper: README.md

The Data Highway's (DHX) Inter-Chain Data Market allows participants to become data providers and to share IoT data from their devices to application developers in exchange for DHX tokens.

#### LPWAN

> A caterory of devices that runs on low bandwidth (less than 125kHz, 200bps), and consumes low power (less than 2W when transmitting). Typicall technologys involves LoraWAN, Sigfox, Weightless, NB-IoT.

Reviewer: ```suggestion
A category of devices that run on low bandwidths (less than 125kHz, 200bps) with low power consumption (less than 2W when transmitting). Typically the technologies involved include LoRaWAN, Sigfox, Weightless, NB-IoT.
```

#### Roaming

> Devices (LPWAN IoT End Devices) are owned by Data Providers and registered at a "home" Network Server (or Supernode that has purchased specific Network ID) that belongs to a specific Network Operator (such as the MXC Network).

Reviewer: ```suggestion
Devices (LPWAN IoT End Devices) are owned by Data Providers and registered at a "home" Network Server (or Supernode that has purchased a specific Network ID) that belongs to a specific Network Operator (such as the MXC Network).
```
